jupyterhub/jupyterhub/base/jupyterhub-singleuser-profiles-configmap.yaml

```diff
@@ -9,6 +9,8 @@ data:
         env:
           - name: S3_ENDPOINT_URL
             value: $(s3_endpoint_url)
+          - name: TRUSTED_CA_BUNDLE_PATH
+            value: $(trusted_ca_bundle_path)
```

Comment:
This should be moved to the overlay and added via a kustomize patch so that the base deployment is not affected by the environment variable

Reply:
I wondered about effects, too. But when I remove the parameter s3_endpoint_url, that env parameter, too, was not present in both jupyterhub and spawned containers anymore. So I figured this approach was ok.

Reply:
moved to the overlay, that is, jupyterhub-singleuser-profiles-configmap in its entirety.